src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource_ProduceTags.cs

```diff
@@ -61,7 +59,7 @@ private void RemoveAllTags()
             var oldTagTree = GetTagTree(snapshot, oldTagTrees);
 
             // everything from old tree is removed.
-            RaiseTagsChanged(snapshot.TextBuffer, new DiffResult(added: null, removed: new(oldTagTree.GetSpans(snapshot).Select(s => s.Span))));
+            RaiseTagsChanged(snapshot.TextBuffer, new DiffResult(added: null, removed: oldTagTree.GetSnapshotSpanCollection(snapshot)));
```

Comment:
two linq calls, and iteration of those, became one direct call to get the NormalizedSpanCollection.